src/components/routes/beforeAuth.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Homepage from '../views/homepage/homepage';
+import LoginLanding from '../views/Login/LoginLanding.jsx';
+import InvestorsSignup from '../views/investors/SignupLanding'
+import InvestorsLogin from '../views/investors/InvestorsLoginLanding'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../views/investors/InvestorsLoginLanding"](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)